it/test/controllers/NextUpdatesControllerISpec.scala

```diff
@@ -354,10 +354,72 @@ class NextUpdatesControllerISpec extends ComponentSpecBase {
           verifyNextUpdatesCall(testNino)
           IncomeTaxViewChangeStub.verifyGetObligations(testNino)
 
-          Then("show Internal Server Error Page")
+          Then("the view displays the correct title even if the OptOut fail")
           res should have(
-            httpStatus(INTERNAL_SERVER_ERROR),
-            pageTitleIndividual("standardError.heading", isErrorPage = true)
+            httpStatus(OK),
+            pageTitleIndividual("nextUpdates.heading")
+          )
+        }
+
+        "Calculation API Failure" in {
+          enable(OptOut)
+
+          val currentTaxYear = TaxYear.forYearEnd(dateService.getCurrentTaxYearEnd)
+          val previousYear = currentTaxYear.addYears(-1)
+
+
+          IncomeTaxViewChangeStub.stubGetIncomeSourceDetailsResponse(testMtditid)(OK, propertyOnlyResponse)
+
+          IncomeTaxViewChangeStub.stubGetNextUpdates(testNino, ObligationsModel(Seq(singleObligationQuarterlyModel(testPropertyIncomeId))))
+
+          IncomeTaxViewChangeStub.stubGetFulfilledObligationsNotFound(testNino)
+          ITSAStatusDetailsStub.stubGetITSAStatusDetails(previousYear.formatTaxYearRange)
+          CalculationListStub.stubGetLegacyCalculationListError(testNino, previousYear.endYear.toString)
```

Comment:
Is it possible to re-phrase this in terms of the Crystallisation Status that we're interested in? Maybe simply wrap this call in something that expresses that. Also, the test name "Calculation API Failure" could change too?